server/etc/e-smith/templates/etc/postfix/main.cf/30smtpauth

```diff
@@ -14,6 +14,9 @@ smtpd_sasl_authenticated_header = yes
 smtpd_tls_cert_file = /etc/pki/dovecot/certs/dovecot.pem
 smtpd_tls_key_file = /etc/pki/dovecot/private/dovecot.pem
 
+# Use specific DH group 2048 bit, renewed each week by cron
+smtpd_tls_dh1024_param_file = /etc/pki/dovecot/private/postfix.dh.param
```

Comment:
Can Postfix start even if the file wasn't created yet?

Reply:
yes it uses the default postfix dh group, then once the new dh group created it is reloaded